pkg/tenant/members.go

```diff
@@ -131,12 +133,29 @@ func (s *Server) TenantMemberCreate(c *gin.Context) {
 //
 // Route: /member
 func (s *Server) MemberList(c *gin.Context) {
-	// TODO: Fetch the member's tenant ID from key.
-	var tenantID ulid.ULID
+	var (
+		err    error
+		member *tokens.Claims
+	)
 
-	// Get members from the database and return a 500 response
-	// if not succesful.
-	if _, err := db.ListMembers(c.Request.Context(), tenantID); err != nil {
+	// Fetch member from the context.
+	if member, err = middleware.GetClaims(c); err != nil {
+		log.Error().Err(err).Msg("could not fetch member from context")
+		c.JSON(http.StatusInternalServerError, api.ErrorResponse("could not fetch member from context"))
+		return
+	}
+
+	// Get the member's orgnaization ID and return a 400 response if it is not a ULID.
```

Comment:
I am wondering if we should return a 401 response here if the `member.OrgID` is empty. Since we are retrieving the `OrgID` from the claims, if the field is empty then it implies that something went wrong with authentication instead of something in the request itself (e.g. there's a middleware bug or the frontend sent a bad token with the request).

Similarly, perhaps if the OrgID is not empty but is not a ULID should we return a 500?

Reply:
I'd debated over whether a 500 would be more appropriate if the OrgID isn't a ULID, so I agree with you here and will update accordingly.